cwms-data-api/src/main/java/cwms/cda/api/TurbineController.java

```diff
@@ -0,0 +1,239 @@
+                            content = {
+                                    @OpenApiContent(type = Formats.JSONV1, from = Turbine.class)
+                            })
+            },
+            description = "Returns CWMS Turbine Data",
+            tags = {TAG}
+    )
+    @Override
+    public void getOne(@NotNull Context ctx, @NotNull String name) {
+        String office = requiredParam(ctx, OFFICE);
+        try (Timer.Context ignored = markAndTime(GET_ONE)) {
+            DSLContext dsl = getDslContext(ctx);
+            TurbineDao dao = new TurbineDao(dsl);
+            Turbine turbine = dao.retrieveTurbine(name, office);
+            String header = ctx.header(Header.ACCEPT);
+            String formatHeader = header != null ? header : Formats.JSONV1;
+            ContentType contentType = Formats.parseHeader(formatHeader, Turbine.class);
+            ctx.contentType(contentType.toString());
+            String serialized = Formats.format(contentType, turbine);
+            ctx.result(serialized);
+            ctx.status(HttpServletResponse.SC_OK);
+            requestResultSize.update(serialized.length());
+        }
+    }
+
+    @OpenApi(
+            requestBody = @OpenApiRequestBody(
+                    content = {
+                            @OpenApiContent(from = Turbine.class, type = Formats.JSONV1)
+                    },
+                    required = true),
+            queryParams = {
+                    @OpenApiParam(name = FAIL_IF_EXISTS, type = Boolean.class,
+                            description = "Create will fail if provided ID already exists. Default: true")
+            },
+            description = "Create CWMS Turbine",
+            method = HttpMethod.POST,
+            tags = {TAG},
+            responses = {
+                    @OpenApiResponse(status = STATUS_204, description = "Turbine successfully stored to CWMS.")
+            }
+    )
+    @Override
+    public void create(Context ctx) {
+        try (Timer.Context ignored = markAndTime(CREATE)) {
+            String acceptHeader = ctx.req.getContentType();
+            String formatHeader = acceptHeader != null ? acceptHeader : Formats.JSONV1;
+            ContentType contentType = Formats.parseHeader(formatHeader, Turbine.class);
+            Turbine turbine = Formats.parseContent(contentType, ctx.body(), Turbine.class);
+            turbine.validate();
+            boolean failIfExists = ctx.queryParamAsClass(FAIL_IF_EXISTS, Boolean.class).getOrDefault(true);
+            DSLContext dsl = getDslContext(ctx);
+            TurbineDao dao = new TurbineDao(dsl);
+            dao.storeTurbine(turbine, failIfExists);
+            ctx.status(HttpServletResponse.SC_CREATED).json("Created Turbine");
+        }
+
+    }
+
+    @OpenApi(
+            pathParams = {
+                    @OpenApiParam(name = NAME, description = "Specifies the name of "
+                            + "the turbine to be deleted."),
```

Comment:
this is update not delete

Reply:
oh  or are you saying that hte name will be deleted b/c its being replaced?  Even if that were so, rewrite that sentence.

Reply:
naw, just a bad copy/paste. Updated